server/src/main/java/io/seata/server/lock/db/DataBaseLocker.java

```diff
@@ -50,8 +51,8 @@ public DataBaseLocker() {
      * @param logStoreDataSource the log store data source
      */
     public DataBaseLocker(DataSource logStoreDataSource) {
-        lockStore = EnhancedServiceLoader.load(LockStore.class, StoreMode.DB.name(), new Class[] {DataSource.class},
-            new Object[] {logStoreDataSource});
+        lockStore = EnhancedServiceLoader.load(LockStore.class, StoreMode.DB.getName(), new Class[] {DataSource.class},
+                new Object[] {logStoreDataSource});
```

Comment:
Should keep format unchanged as before, redundant tab character here.

Reply:
Ok,removed the redundant tab character.